docker-php-source

```diff
@@ -19,7 +19,7 @@ case "$1" in
 			echo >&2 "$dir exists and is not a directory"
 			exit 1
 		fi
-		if [ ! -d "$dir" ]; then
+        if [[ ! -d "$dir" || $(ls -1 "$dir") == "ext" ]]; then
```

Comment:
I just don't check that `ext` is a directory here... but I think we shouldn't care because people in this case would intentionally make their builds fail I guess ^^